pkg/TIPA.py

```
@@ -138,14 +189,18 @@ class Worker(Process):
 
     Does stuff it's told to do in the queue.
     '''
-    def __init__(self, queue, lock):
-        super(Worker, self).__init__()
+    def __init__(self, queue, lock, name="WorkerProcess"):
```

Ruff (0.8.0)

Missing return type annotation for special method `__init__`

Add return type annotation: `None`

(ANN204)